projects/layout/components/search/search-filters.template.html

```diff
@@ -8,9 +8,11 @@
     #rem
     class="t-rem"
 ></div>
+<!-- TODO: title i18n -->
 <button
     *ngIf="overflown() < templates.length"
     appearance="secondary"
+    title="Filters"
```

Comment:
Should be as plain text here:
https://github.com/taiga-family/taiga-ui/blob/main/projects/layout/components/search/search-filter.component.ts#L21